train.py

```diff
@@ -39,6 +39,8 @@
 flags.DEFINE_string('export_path', None,
                     'Where to export the model after training.')
 
+flags.DEFINE_bool('use_bt', False, 'Whether to use Bigtable as input.')
```

Comment:
can you add something like "only supported with TPU currently"

Reply:
Done!